Message Orchestration/Canvas Sync Alt Text (#3293)
braze-inc/braze-docs: _docs/_partners/message_orchestration/additional_channels/messaging/line.md

Next, paste your channel access token in the Content Block body, and save it.

![An image the Content Block showing the Content Block name, Liquid tag, and censored channel access token.][2]

Once you've set the channel access token inside a Content Block, you will be able to use the Line webhook template to send messages to users.

### Step 4: Select a webhook template

From **Templates & Media**, go to **Webhook Templates** and choose one of the following Line Messenger webhook templates: 

> ![An image of some of the available predesigned webhook templates in Braze.]({% image_buster /assets/img_archive/line_templates.png %}){: style="border:0px;"}

Reviewer: ```suggestion
![A selection of available predesigned webhook templates.]({% image_buster /assets/img_archive/line_templates.png %}){: style="border:0px;"}
```

Before you send your message, test your webhook. Make sure your Line ID is saved in Braze (or find it and test as a customized user), and use the preview to send the test message:

![An image of the test tab in the Braze webhook builder showing you can preview the message by sending it to an existing user.][3]

If you receive the message successfully, you can start to configure its delivery settings.

## Using this integration

Once set up, use this integration to target Line users. First, [create a segment][62] for all users for whom the `Line ID` exists as a custom attribute and turn on [analytics tracking][61] to track your Messenger subscription rates over time. 

> ![An image of the segment filter "line_id" set to "is not blank".][63]

Reviewer: ```suggestion
![Segment filter "line_id" set to "is not blank".][63]
```
